packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Update compileSdkVersion to 33.
```

Comment:
I don't believe this :)

Reply:
hmm this was automatically added by the change tool. Will revert.